osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -101,10 +114,11 @@ public abstract class DrawableHitObject : SkinReloadableDrawable
         protected DrawableHitObject([NotNull] HitObject hitObject)
         {
             HitObject = hitObject ?? throw new ArgumentNullException(nameof(hitObject));
+            positionalSoundAdjustment.Value = PositionalSound;
```

Comment:
Will this work properly in case of mods that move hitobjects around after they've been created? It looks like this only uses the initial position and doesn't consider any changes later on. 

I know it's only `Wiggle` for now - which won't really cause any noticeable differences here - but more extreme mods might come in the future where this might become a problem.

Reply:
You're right, and now that you mention it I see that I slightly deviated from peppy's expression of the PositionalSound. The value considered should indeed be the position of the drawable and not the one specified in the beatmap. I will try implementing this behaviour.